NativeApp/include/Android/AndroidAppBase.hpp

```diff
@@ -81,15 +84,39 @@ class AndroidAppBase : public AppBase
     android_app* app_ = nullptr;
     std::string  native_activity_class_name_;
 
+    pthread_mutex_t mutex;
```

Comment:
It is better to use std::mutex

Reply:
ok, though i think std::mutex is slightly slower than pthread_mutex in terms of overhead, but i will switch to std::mutex